clients/runtime/src/types.rs

```diff
@@ -214,7 +214,10 @@ mod vault_id {
 
 mod dispatch_error {
 	use crate::metadata::{
-		runtime_types::sp_runtime::{ArithmeticError, ModuleError, TokenError, TransactionalError},
+		runtime_types::{
+			sp_arithmetic::ArithmeticError as OtherArithmeticError,
```

Comment:
Why can't we leave the name as 'ArithmeticError'? Why does it need to be 'OtherArithmeticError'?

Reply:
Oh. Sure . You are right. Now we can. Fixed.
There was a line with the import of other ArithmeticError.
I removed that line and now there is no need for alias.
Thanks. FIxed.